CHANGES.md

```diff
@@ -8,6 +8,8 @@
 
 - Support folding of `ifthenelse` expressions (#1031)
 
+- Add `--context` flag (#1238)
```

Comment:
I think we could add a bit more context here at least define what "context" means here.